stories/index.js

```diff
@@ -22,6 +23,9 @@ storiesOf('NewsItem', module)
   .add('not upvoted', () => (
     <NewsItem upvoted={false} />
   ))
+  .add('with 12 comments', () => (
+    <NewsItem commentCount={12} />
```

Comment:
Does the NewsItem have to have an `upvoted` prop? I think it was set as required in the `NewsItem.propTypes`?